README.md

```diff
@@ -50,74 +51,92 @@ to master and using the Github Action.
 
 **Run SourceCred**
+
 - `yarn start` creates the cred graph, computes cred scores and runs the front end interface which you can access at `localhost:6006`
-in your browser.
+  in your browser.
 
 NOTE: this command will not load any new data from Discord / GitHub / Discourse, etc. If you want to re-load
 all the latest user activity, run `yarn load` again.
 
 **Clear Cache**
 
-
 - `yarn clean` will clear any cached data that was loaded by the plugins. You can run this if any plugins fail to load. Run `yarn load` after this to re-load the data.
 
 If you want to restart from a clean slate and remove all the generated graphs, you can do so via:
-- `yarn clean-all` 
+
+- `yarn clean-all`
 
 Run `yarn clean-all` if the `yarn start` command fails due to a change in the config or breaking changes in a new version of SourceCred.
 
 ### Distributing Grain
-- `yarn grain` distributes Grain according to the current Cred scores, and the config in `config/grain.json`. 
+
+- `yarn grain` distributes Grain according to the current Cred scores, and the config in `config/grain.json`.
 
 This repo also contains a GitHub action for automatically distributing grain. It will run every Sunday and create a Pull Request
 with the ledger updated with the new grain balances based on the users Cred scores. The amount of grain to get distributed
-every week can be defined in the `config/grain.json` file. 
-
+every week can be defined in the `config/grain.json` file.
 
 There are three different policies that can be used to control
-how the grain gets distributed: `immediatePerWeek`, `balancedPerWeek`, and `recentPerWeek`. For info on what each policy does, how to choose the right policy for your community, and how Grain operates in general, see [How Grain Works](https://sourcecred.io/docs/beta/grain). 
+how the grain gets distributed: `immediatePerWeek`, `balancedPerWeek`, and `recentPerWeek`. For info on what each policy does, how to choose the right policy for your community, and how Grain operates in general, see [How Grain Works](https://sourcecred.io/docs/beta/grain).
 
-Below is an example `grain.json` file for a configuration that uses a combination of all three policies. Here we tell SourceCred to distribute 1,000 grain every week, with 25% (250 grain) distributed according to `immediatePerWeek`, 25% (250 grain) distributed according to `balancedPerWeek`, and 50% (500 grain) distributed according to `recentPerWeek`. 
+Below is an example `grain.json` file for a configuration that uses a combination of all three policies. Here we tell SourceCred to distribute 1,000 grain every week, with 25% (250 grain) distributed according to `immediatePerWeek`, 25% (250 grain) distributed according to `balancedPerWeek`, and 50% (500 grain) distributed according to `recentPerWeek`.
 
-```
+```json
 {
- "immediatePerWeek": 250,
- "balancedPerWeek": 250,
- "recentPerWeek": 500,
- "recentWeeklyDecayRate": 0.5,
  "maxSimultaneousDistributions": 100
+ "allocationPolicies": [
```

Comment:
I would only ask for you to add some docs for the SPECIAL distribution as well but it's all good if it was meant to make these docs correct

Reply:
I don't think its really advisable to create `SPECIAL` distributions via the grain config file. There is a UI route that is much less clumsy and heavy-handed.